index.bs

```diff
@@ -2272,6 +2264,103 @@ Authenticators:
 - should ensure that the [=signature counter=] value does not 
     accidentally decrease  (e.g., due to hardware failures).
 
+
+## Authenticator taxonomy ## {#sctn-authenticator-taxonomy}
+
+Different kinds of [=authenticators=] enable different use cases along two independent dimensions: the [=authentication ceremony
+structure=] and the [=authenticator=]'s [=attachment modality=]. The [figure below](#fig-authData) illustrates how the use cases
+relate to different kinds of [=authenticators=].
+
+<figure id="fig-authenticator-taxonomy">
```

Comment:
Why does the upper circle contain User Verification AND First Factor?  According to the description of First Factor, only the term First Factor should be sufficient (and less confusing).

Similarly the lower circle should only contain username-less.

UNLESS: another circle is added to just contain User Verification that is fully included in the upper circle.
Similarly for the lower circle and the term Client-side c.p.k..

Reply:
Good points. My intention was to more clearly relate those authenticator properties to the use cases they enable, but you're right it probably just makes it more cluttered.